generators/server/index.js

```diff
@@ -154,6 +154,11 @@ module.exports = JhipsterServerGenerator.extend({
                 this.messageBroker = false;
             }
 
+            this.apiFirst = this.config.get('apiFirst') === 'no' ? false : this.config.get('apiFirst');
```

Comment:
Just to be sure : do I keep a string option in `.yo-rc.json` that is transformed to a boolean flag for the templates ?